config/smtpConfig/SmtpConfigTest.groovy

```diff
@@ -84,4 +84,16 @@ class SmtpConfigTest extends Specification {
         conn.outputStream.write(backup)
         conn.disconnect()
     }
+
+    def matchesSmtpConfiguration(json1, json2) {
```

Comment:
This method do not compare passwords because for artifactory 5.5.x password encryption is enabled by default.